DBMS/DBMS-Session3/assignment4_sql.txt

```diff
@@ -0,0 +1,30 @@
+SHOW DATABASES;
+USE ShopCart;
+SHOW TABLES;
+
+CREATE TABLE zip_code_list(
```

Comment:
Create separate tables for state city and code zip for a city is specific for that city.